src/main/kotlin/app/revanced/util/fingerprint/FingerprintUtils.kt

```diff
@@ -0,0 +1,23 @@
+package app.revanced.util.fingerprint
+
+import org.jf.dexlib2.Opcode
+import org.jf.dexlib2.iface.ClassDef
+import org.jf.dexlib2.iface.Method
+import org.jf.dexlib2.iface.instruction.Instruction
+
+object FingerprintUtils {
+    // TODO: Use CustomFingerprint typealias once https://github.com/revanced/revanced-patcher/pull/189 is merged
+    inline fun <reified T : Instruction> any(
+        opcode: Opcode,
```

Comment:
Any reason why this is required?

Reply:
For example, both `invoke virtual` and `invoke static` can be a method reference instruction.

Reply:
You could use `reified T : ReferenceInstruction`

Reply:
Yes but then the method wouldn't work for other `Instruction`s - shouldn't this be generic? I just used method reference instructions because I had a usecase for them (and I don't really know much about the other instructions), but ideally, there would be an extension method for each instruction in `InstructionExtensions` (and a dedicated fingerprint class for each instruction)